services/movie.py

```diff
@@ -25,16 +32,17 @@ def get_movie_by_id(movie_id: int) -> Movie:
 def create_movie(
     movie_title: str,
     movie_description: str,
-    genres_ids: list = None,
-    actors_ids: list = None,
+    genres_ids: Optional[list] = None,
+    actors_ids: Optional[list] = None,
 ) -> Movie:
-    movie = Movie.objects.create(
-        title=movie_title,
-        description=movie_description,
-    )
-    if genres_ids:
-        movie.genres.set(genres_ids)
-    if actors_ids:
-        movie.actors.set(actors_ids)
+    with transaction.atomic():
```

Comment:
Please, use the decorator `transaction.atomic` instead of context manager.
[https://docs.djangoproject.com/en/4.2/topics/db/transactions/#controlling-transactions-explicitly](https://docs.djangoproject.com/en/4.2/topics/db/transactions/#controlling-transactions-explicitly)